tests/settings/SettingTest.py

```diff
@@ -36,6 +42,15 @@ def test_path_list(self):
         self.assertEqual(path_list(self.uut),
                          [os.path.abspath(os.path.join("test", ".")), abspath])
 
+    def test_glob_list(self):
+        abspath = os.path.abspath(".")
+        # Need to escape backslashes since we use list conversion
+        self.uut = Setting("key", "., " + abspath.replace("\\", "\\\\"),
```

Comment:
I just wrote something similar to what is in `test_path_list`. I'll make a newcomer issue solving both or do I make the change myself?